annotations.md

```diff
@@ -23,6 +23,7 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.url** URL to find more information on the image (string)
 * **org.opencontainers.image.documentation** URL to get documentation on the image (string)
 * **org.opencontainers.image.source** URL to get source code for building the image (string)
+* **org.opencontainers.image.source.subpath** A tool-specific path within the `source` repository which be used to distinguish different build targets in the same source repository
```

Comment:
A tool-specific path should be just a third party annotation with their reversed-domain prefix

Reply:
And probably it doesn’t need to be a tool-specific:

```suggestion
* **org.opencontainers.image.source.subpath** A relative path within the `source` repository which be used to distinguish different build targets in the same source repository. The path separator MUST be `slash`. The path MUST NOT contain “..” (upper dir) component.
```

Reply:
Does the path separator need to be `/` even on Windows systems (genuinely don't know enough to answer here).